examples/torque_driven_ocp/track_markers_2D_pendulum.py

```diff
@@ -151,10 +151,10 @@ def main():
 
     biorbd_path = str(EXAMPLES_FOLDER) + "/getting_started/pendulum.bioMod"
     biorbd_model = biorbd.Model(biorbd_path)
-    final_time = 3
+    final_time = 1
     n_shooting = 20
 
-    ocp_to_track = data_to_track.prepare_ocp(biorbd_model_path=biorbd_path, final_time=3, n_shooting=19)
+    ocp_to_track = data_to_track.prepare_ocp(biorbd_model_path=biorbd_path, final_time=1, n_shooting=19)
```

Comment:
Either remove the `final_time` variable or use it in the `final_time=final_time`, same for `n_shooting`